src/Workspaces/CSharp/Portable/Classification/ClassificationHelpers.cs

```diff
@@ -159,12 +268,40 @@ private static string GetClassificationForIdentifier(SyntaxToken token)
             {
                 return ClassificationTypeNames.Keyword;
             }
+            else if (token.Parent is ExternAliasDirectiveSyntax externAliasDirectiveSyntax && externAliasDirectiveSyntax.Identifier == token)
+            {
+                return ClassificationTypeNames.NamespaceName;
+            }
+            else if (token.Parent is LabeledStatementSyntax labledStatementSyntax && labledStatementSyntax.Identifier == token)
+            {
+                return ClassificationTypeNames.LabelName;
+            }
             else
             {
                 return ClassificationTypeNames.Identifier;
             }
         }
 
+        public static bool IsStaticallyDeclared(SyntaxToken token)
+        {
+            var parentNode = token.Parent;
+
+            if (parentNode.IsKind(SyntaxKind.EnumMemberDeclaration))
+            {
+                return false; // TODO: Since Enum members are always static is it useful to classify them as static?
+            }
+            else if (parentNode.IsKind(SyntaxKind.VariableDeclarator))
+            {
+                // The parent of a VariableDeclartor is a VariableDeclarationSyntax node.
+                // It's parent will be the declaration syntax node.
+                parentNode = parentNode.Parent.Parent;
+
+                return parentNode.GetModifiers().Any(modifier => modifier.IsKind(SyntaxKind.StaticKeyword, SyntaxKind.ConstKeyword));
+            }
```

Comment:
i'm curious if we should think of consts as statics.   I can see arguments both ways.  What's the core use case for 'static'? to colorize differently?  or do do something like make the code italicized?

Reply:
The idea is that the additive classifications would change the font style, although we are limited to Bold at the moment.

Reply:
Bold and Italic right?  Italic would actually make a ton of sense to me.  Bold is a bit too offputting.

Reply:
You can set Italic from the MEF exported FormatDefinition but there is no way to override it in the Fonts and Colors option page.